ci/scripts/e2e-cassandra-sink-test.sh

```diff
@@ -68,6 +62,16 @@ else
   exit 1
 fi
 
+if cat ./query_result2.csv | awk -F "," '{
+    exit !($1 == 1 && $2 == 1 && $3 == "1\r"); }'; then
+  echo "Cassandra sink check passed"
+else
+  echo "The output is not as expected."
+  echo "output:"
+  cat ./query_result2.csv
+  exit 1
+fi
```

Comment:
nit: I think we can also put the validation logic in `cassandra_sink.slt` using `system` command

Reply:
okk